src/cli/commands/public-cmds/add-cmd.js

```diff
@@ -48,6 +49,11 @@ export default class Add extends Command {
     const exludedFiles = exclude
       ? this.splitList(exclude).map(filePath => pathNormalizeToLinux(filePath.trim()))
       : undefined;
+
+    // check if user is trying to add test files only without id
+    if (!R.isEmpty(tests) && !id && R.isEmpty(normalizedPathes)) {
+      return Promise.reject("You can't add test files without specifying component id");
```

Comment:
please create new Error, it's bad to use reject with a string.
(I know there is a lot of places we do it with strings, we need to fix them all)